pkg/newlog/go.mod

```diff
@@ -1,14 +1,13 @@
 module github.com/lf-edge/eve/pkg/newlog
 
-go 1.23
-toolchain go1.24.1
```

Comment:
You don't need to remove the toolchain line anymore, sources parser was updated here: https://github.com/lf-edge/eve/commit/6c40145be52f9c4259da98d1575202edc10da711

Reply:
Added :+1: